tensorflow_serving/util/class_registration.h

```diff
@@ -266,7 +266,9 @@ class ClassRegistry {
         protobuf::MessageFactory::generated_factory()
             ->GetPrototype(descriptor)
             ->New());
-    any_config.UnpackTo(config.get());
+    if (!any_config.UnpackTo(config.get())) {
+      return errors::InvalidArgument("Malformed content of Any.value");
```

Comment:
It would be helpful to print the malformed thing here, if possible. Would something like this work:

return errors::InvalidArgument(...) << any_config.DebugString();

Reply:
Thanks, I changed the error message to include the `DebugString`.